src/sentry/dynamic_sampling/rules/utils.py

```diff
@@ -180,3 +182,8 @@ def apply_dynamic_factor(base_sample_rate: float, x: float) -> float:
         )
 
     return float(x / x**base_sample_rate)
+
+
+def get_redis_client_for_ds() -> Any:
+    cluster_key = getattr(settings, "SENTRY_DYNAMIC_SAMPLING_RULES_REDIS_CLUSTER", "default")
```

Comment:
If you're storing data for all projects at a single point in time, you almost certainly want to provision a new cluster for this. It can be a single memorystore instance. Alternatively it may be time to try out @beezz new RC cluster setup

Reply:
we want to reuse the same dedicated cluster which we use for another Dynamic Sampling bias.

Reply:
it's ok then, just don't use `default`